CardSign.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace cardGameSix
+{
+    public enum Sign : int { HEARTS = 1, DIAMONDS = 2, SPADES = 3, CLUBS = 4 };
```

Comment:
I think the name `Sign` should be the same as the file name a`CardSign` - use refactor (rename field) https://docs.microsoft.com/en-us/visualstudio/ide/reference/rename?view=vs-2019 here and below

```suggestion
    public enum CardSign : int { HEARTS = 1, DIAMONDS = 2, SPADES = 3, CLUBS = 4 };
```